### ISSUE TITLE
Accessibility Issue - Inappropriate usage of aria-hidden on focusable elements

### ISSUE THREAD

**Issue by Nathanael-ms** [microsoft/BotFramework-WebChat#4652]
### Is it an issue related to Adaptive Cards?

No.

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

I am not using latest

### Which area does this issue affect?

Focus management

### How to reproduce the issue?

1. Navigate to ...
2. Send at least one message.
3. Open Dev Tools and inspect the first `article` element contained in the transcript section.
4. Observe that the `article` element contains direct descendants with both `aria-hidden="true"` and `tabindex="0"`.
<img width="435" alt="image" src="https://user-images.githubusercontent.com/126293949/221322755-1ad7d64f-dfdf-4dad-96a4-a40c8608136c.png">

### What do you expect?

Elements with `aria-hidden="true"` should not be focusable.

### What actually happened?

The DOM included focusable elements with `aria-hidden="true"`.

### Did you find any DOM elements that might have caused the issue?

The issue seems to appear in the FocusRedirector component:
https://github.com/microsoft/BotFramework-WebChat/blob/v4.15.7/packages/component/src/Utils/FocusRedirector.tsx

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

### Additional context

This issue was detected using Accessibility Insights for Web. More info on the rule that led to the violation can be found at the following link:
https://accessibilityinsights.io/info-examples/web/aria-hidden-focus/

**Comment by compulim** [comment]
Reopened for pending verification.

This is fixed in 4.15.7.main-573cfb7.

**Comment by compulim** [comment]
This is fixed and released in `4.15.8`.
